pkg/controllers/dynakube/logmonitoring/daemonset/env.go

```diff
@@ -65,19 +65,25 @@ func getInitEnvs(dk dynakube.DynaKube) []corev1.EnvVar {
 			Name:  clusterUIDEnv,
 			Value: dk.Status.KubeSystemUUID,
 		},
-		{
-			Name:  clusterNameEnv,
-			Value: dk.Status.KubernetesClusterName,
-		},
-		{
-			Name:  entityEnv,
-			Value: dk.Status.KubernetesClusterMEID,
-		},
 		{
 			Name:  basePodNameEnv,
 			Value: dk.LogMonitoring().GetDaemonSetName(),
 		},
 	}
+
+	if dk.Status.KubernetesClusterMEID != "" && dk.Status.KubernetesClusterName != "" {
+		envs = append(envs, corev1.EnvVar{
```

Comment:
append support multi value append, nitpick but can be refactored as:
```
package main

import "fmt"

func main() {
	a := []int{}

	a = append(a, 1, 2)

	fmt.Println(a)
}
```
envs  = append(envs, corev1.EnvVar{}, corev1.EnvVar{})
```

```

Reply:
done https://github.com/Dynatrace/dynatrace-operator/pull/5239/commits/4a3b7d1fd86a7edc826cceaf61f1e03c1b5cdc86